core/src/css/link.ionic.scss

```diff
@@ -0,0 +1,120 @@
+@import "../foundations/ionic.vars";
+
+// Link: Shared Styles (Standalone & Underline)
+// -------------------------------------------------------------------------------
+@mixin link-shared {
+  display: inline-flex;
+
+  align-items: center;
+
+  gap: 4px;
+
+  transition: color 0.2s ease-in-out;
+
+  font-weight: 400;
+
+  text-decoration-color: inherit;
+
+  text-underline-offset: 2px;
+
+  cursor: pointer;
+
+  // Link: Visited
+  // -------------------------------------------------------------------------------
+
+  &:visited {
+    color: $ionic-color-info-500;
+  }
+}
+
+// Link: Standalone
+// -------------------------------------------------------------------------------
+
+@mixin ion-link {
+  @include link-shared;
+
+  // Link: Standalone - Hover
+  // -------------------------------------------------------------------------------
+
+  @media (any-hover: hover) {
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  color: $ionic-color-info-400;
```

Comment:
Did design say to use the info color here? The design requirements state:

> 	By default, the link color is the primary color. But developers should be able to change it.

Reply:
Implementation is based on Figma, which is marked as dev-ready and updated. Design will have a final opportunity to test the style implementation in ODC.

The UX requirements are already inconsistent/incorrect:

![image](https://github.com/ionic-team/ionic-framework/assets/13732623/fb8de681-feb6-413e-a834-d27b75b72e98)

References the info colors used/in Figma, but the focus outline is wrong. I will bring this up with the larger Design team.

We do the best with the information and accuracy of the information given 🙂

Reply:
Sounds good 👍